products/nvidiadriver.md

```diff
@@ -156,18 +196,18 @@ releases:
     releaseDate: 2020-06-24
     eoas: 2020-12-15
     eol: 2023-07-01
-    latest: "453.94"
-    latestReleaseDate: 2022-11-22
-    link: https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-450-216-04/
+    latest: "454.23"
+    latestReleaseDate: 2023-06-26
+    link: https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-450-248-02/
 
 -   releaseCycle: "R450-Linux"
     lts: true
     releaseDate: 2020-06-24
     eoas: 2020-10-07
     eol: 2023-07-01
     latest: "450.216.04"
```

Comment:
```suggestion
    latest: "450.248.02"
```

Reply:
Looks like I don't have the permission to commit this change. Can you do it and let me know @rcalixte ?

Reply:
Done! Apologies for the oversight!